src/darwin/Framework/CHIP/MatterControllerFactory.mm

```diff
@@ -196,7 +195,11 @@ - (BOOL)startup:(MatterControllerFactoryParams *)startupParams
 
         // Initialize device attestation verifier
         if (startupParams.paaCerts) {
-            _attestationTrustStoreBridge->Init(startupParams.paaCerts);
+            _attestationTrustStoreBridge = new CHIPAttestationTrustStoreBridge(startupParams.paaCerts);
+            if (_attestationTrustStoreBridge == nullptr) {
```

Comment:
Should you call `self checkForInitError:(_attestationTrustStoreBridge != nullptr)` here?

Reply:
No, because we're not inside `init`.  That would clean up various things that `init` allocates, which we very much do not want to do here.

Instead we just return early, don't set ourselves as running, and clean up stuff from just this method at the end of the method.